app/ui/src/main/java/com/fsck/k9/fragment/MessageListFragment.java

```diff
@@ -169,7 +169,6 @@ public static MessageListFragment newInstance(
     private MessageListAdapter adapter;
     private View footerView;
     private FolderInfoHolder currentFolder;
-    private LayoutInflater layoutInflater;
```

Comment:
Why remove the field and look up the `LayoutInflater` every time one is needed?

Reply:
I'll look it up again, but I believe the only reason for this field and a getter exit is for the list adapter.

I'll leave it as a field if we do  another service location.